packages/commonwealth/client/scripts/views/components/SublayoutHeader/DesktopHeader/UserDropdown/UserDropdown.tsx

```diff
@@ -9,6 +8,7 @@ import { User } from 'views/components/user/user';
 
 import useUserMenuItems from '../../useUserMenuItems';
 
+import useUserStore from 'client/scripts/state/ui/user';
```

Comment:
i noticed that you always use `client/scripts` prefix whereas `state/ui/user` is enough. Not a big deal but I think vite is configured that way so we don't need to write those long imports.

Reply:
oh, these are just auto imports adding that (now sure why), will cleanup